rest-api-spec/src/main/resources/rest-api-spec/test/tasks.list/10_basic.yml

```diff
@@ -17,8 +17,14 @@
   - do:
       tasks.list:
         group_by: parents
+        detailed: true
+  - set:
+      tasks._arbitrary_key_: task_id
 
   - is_true: tasks
+  - is_true: tasks.$task_id.resource_stats
+  - is_true: tasks.$task_id.resource_stats.thread_info
```

Comment:
I added a separate test that checks the thread_info portion, going to leave the resource_stats check in this one too though since that seems like a good addition.